scripts/init.sh

```diff
@@ -28,6 +28,12 @@ HOST_IP=$(curl -H "X-aws-ec2-metadata-token: $TOKEN" http://169.254.169.254/late
 PRIMARY_IF=$(ip -4 -o a | grep "$HOST_IP" | awk '{print $2}')
 sysctl -w "net.ipv4.conf.$PRIMARY_IF.rp_filter=2"
 
+# Configure tcp_early_demux to enable kubelet to pod-eni communication
+# https://lwn.net/Articles/503420/ for background
+if ! [[ -z "$ENABLE_POD_ENI" ]]; then
+    sysctl -w "net.ipv4.tcp_early_demux=0"
+fi
```

Comment:
How about:
```
if [[ "$ENABLE_POD_ENI" == "true" ]]; then
   sysctl -w "net.ipv4.tcp_early_demux=0 "
fi
```

Reply:
🤦  The env variable is not available in the initContainer... 🤔  How about something like:
```
# Disable early mux for tcp by default because it interferes with health checks when using per pod security groups
: "${DISABLE_EARLY_DEMUX:=true}"
if [[ "$DISABLE_EARLY_DEMUX" == "true" ]]; then
   sysctl -w "net.ipv4.tcp_early_demux=0"
   echo "Set net.ipv4.tcp_early_demux=0"
fi
```